docs/content/en/docs/workflows/debug.md

```diff
@@ -405,16 +413,52 @@ WARN[0005] Image "image-name" not configured for debugging: unable to determine
 
 See the language runtime section details on how container images are recognized.
 
+### Why aren't my breakpoints being hit?
+
+**Missing required language extension.**
+Go, Python, and JavaScript debugging in IDEA Ultimate requires the
+corresponding language plugin to be installed. If it is not installed,
+the debugger will not attach and breakpoints will not be hit.
+
+**Breakpoints on startup path are not hit.**
```

Comment:
What do you mean by startup path here?

Reply:
I meant the container execution initialization path (e.g., from main() to listening)

Reply:
Retitled: let me know if that's still unclear.